gamemodes/guesswho/gamemode/sh_config.lua

```diff
@@ -22,6 +22,10 @@ GM.Models = {
     "models/player/Group01/Female_03.mdl",
 }
 
+GM.PropFreezeEnabled = false -- if set to true, world props wont move.(default: false)
```

Comment:
Please make PropFreezeEnabled a convar, by adding an `entry to gamemodes/guesswho/guesswho.txt` not via lua.